browser-tests/utils/settings.ts

```diff
@@ -1,7 +1,6 @@
 // @ts-ignore
 require('dotenv').config({ path: '.env.development.local' });
 
-const TEST_ENV_URL = 'https://venepaikka-admin.test.kuva.hel.ninja';
 const LOCAL_ENV_URL = 'http://localhost:3000';
```

Comment:
Maybe we should allow setting this in `.env.development.local`?

Reply:
Done!